db/migrate/20160205194153_add_omniauth_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddOmniauthToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :provider, :string
+    add_column :users, :uid, :string
```

Comment:
These columns are not used at the moment, but if account creation is implemented I think they will be useful ;-)